fpl/cli.py

```diff
@@ -1,7 +1,7 @@
 """Cli module."""
```

Comment:
Do we want to have one common file for all CLIs? Wouldn't it be better to have one CLI folder with different files with appropriate names to make it easier to use?

Reply:
I don't have strong opinions around how to structure CLI's. 

I think that in this case it would be OK to have one CLI (with the invoke prefex "fantasy") followed by the different cli commands (e.g. "download" or "csv_on_entity") with belonging parameters. It would be a bit overengineering to divide the cli as of now. Perhaps it can later be divided into cli-domains for "data ingestion" / "data visualization" / "machine learning" / ... etc. at a later point?